docs/about.html

```diff
@@ -0,0 +1,44 @@
+
```

Comment:
The indentation is little confusing. 

Don't bother if this file is generates with a tool. Otherwise, it would nice to fix. In emacs I would just mark the whole buffer and run M-x `indent-region`.

Reply:
Thanks! Yes the files are generated programmatically, but I can work on the indentation later when we go thorough a polishing round of the code.